src/runner.js

```diff
@@ -4,13 +4,27 @@ const parser = require('./parser/parser.js');
 const results = require('./results.js');
 const DefaultVariableStorage = require('./default-variable-storage.js');
 const nodeTypes = require('./parser/nodes.js').types;
+const MessageFormat = require('messageformat');
 
 class Runner {
-  constructor() {
+  /**
+   * Creates an instance of Runner.
+   * @param {object} [options] Options for the runner
+   * @param {object} options.formaterOptions Options for messageFormat formater
+   * @param {string} options.formaterOptions.language Options for the language of the formater
+   * @memberof Runner
+   */
+  constructor(options) {
     this.yarnNodes = {};
     this.variables = new DefaultVariableStorage();
     this.functions = {};
     this.visited = {}; // Which nodes have been visited
+    this.options = {
+      formaterOptions: {
```

Comment:
you never seem to use this.options,so I wonder why nest the language like this and spread options outside of formaterOptions

Reply:
I don't really know much about messageFormat library so I was thinking for the future if needed to expend options for other properties

Reply:
ah ok, fair enough. I cant seem to find anything in the documentation where anything but language can be passed
http://messageformat.github.io/messageformat/guide/

if thats true maybe we can just have language without nesting it?